CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project will be documented in this file.
 
+## 1.6.1
+
+### Bug Fixes
+
+- Support textDocument/formatting response with granular text edits ([#148](https://github.com/biomejs/biome-intellij/pull/148))
+- Run text edits in a reversed order ([#151](https://github.com/biomejs/biome-intellij/pull/151))
+- Avoid conflicting code actions on save ([#149](https://github.com/biomejs/biome-intellij/pull/149))
+
+### Features
+
+- Monorepo support ([#138](https://github.com/biomejs/biome-intellij/pull/138))
```

Comment:
I'd prefer 1.7.0 as this is a big change.

Reply:
Hm,
@ematipico could you please help us to delete 1.6.1?🤞🏽

Reply:
FYI - i already got asked to update to 1.6.1 by Intellij

![image](https://github.com/user-attachments/assets/0a84f74c-da36-48e3-905c-367e402a0c8d)

Reply:
Sure I will do that

Reply:
Done, the version should have been deleted

Reply:
Looks good :) Intellij wants to go from 1.6.0 to 1.7.0 and 1.6.1 is gone

https://plugins.jetbrains.com/plugin/22761-biome/versions